README.md

```diff
@@ -20,6 +20,9 @@ From source:
 git clone https://github.com/willthames/ansible-lint
 export PYTHONPATH=$PYTHONPATH:`pwd`/ansible-lint/lib
 export PATH=$PATH:`pwd`/ansible-lint/bin
+cd ansible-lint
+mkdir bin
+ln -s ../lib/ansiblelint/main/__init__.py bin/ansible-lint
```

Comment:
IMO, this is the wrong approach, has it now makes your git repo dirty.  I think the proper way to do this is install ansible-lint into a virtualenv:

```
pip install virtualenv
virtualenv ansible-lint-venv
./ansible-lint-venv/pip install ./ansible-lint
./ansible-lint-venv/bin/ansible-lint
```

more commands, but now you have compiled version of ansible-lint installed into sandboxed virtualenv